docs/cli/notifications_pause.md

```diff
@@ -0,0 +1,11 @@
+<!-- DO NOT EDIT | GENERATED CONTENT -->
+
+# notifications pause
+
+Pause notifications
```

Comment:
It might be good to have a little more explanation about what this does and why an operator might want to use it.

Reply:
Added an extra sentence about potential outage.